### PR TITLE
[16.0][MIG] fieldservice_account

### PR DESCRIPTION
**Pull request by epanisset**

### CODE REVIEW

**Review by yankinmax — APPROVED** [review]
Seems legit, thanks

**Review by ghost — APPROVED** [review]

**Review by SilvioC2C — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

File: fieldservice_account/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    def _compute_fsm_order_ids(self):
+        for order in self:
+            orders = self.env["fsm.order"].search(
+                [("invoice_lines", "in", order.line_ids.ids)]
+            )
+            order.fsm_order_ids = orders
+            order.fsm_order_count = len(order.fsm_order_ids)
+
+    def action_view_fsm_orders(self):
+        action = self.env.ref("fieldservice.action_fsm_dash_order").read()[0]
```
Comment (ghost) [inline fieldservice_account/models/account_move.py]: use the odoo method _for_xml_id it is safer
```suggestion
        action = self.env["ir.actions.act_window"]._for_xml_id(
            "fieldservice.action_fsm_dash_order"
        )
```

File: fieldservice_account/models/fsm_order.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    def _compute_get_invoiced(self):
+        for order in self:
+            invoices = order.invoice_lines.mapped("move_id").filtered(
+                lambda r: r.move_type in ("out_invoice", "out_refund")
+            )
+            order.invoice_ids = invoices
+            order.invoice_count = len(invoices)
+
+    def action_view_invoices(self):
+        action = self.env.ref("account.action_move_out_invoice_type").read()[0]
```
Comment (ghost) [inline fieldservice_account/models/fsm_order.py]: samething as above

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by gurneyalex** [comment]
duplicate #985 which is being merged

**Pull request closed without merging**